data/brands/shop/mobile_phone.json

```
@@ -10,7 +10,7 @@
   },
   "items": [
     {
-      "displayName": "3 Store",
+      "displayName": "3",
       "id": "3store-e135a8",
       "locationSet": {"include": ["gb", "it"]},
       "matchNames": ["3", "three", "three store"],
```

To match existing 3 Stores to this modified entry
```suggestion
       "matchNames": ["3 store", "three", "three store"],
```